renderer/components/preferences/categories/plugins/index.js

```diff
@@ -0,0 +1,187 @@
+    pluginsFromNpm: []
+  }
+
+  render() {
+    const {
+      pluginsInstalled,
+      pluginsFromNpm,
+      pluginBeingInstalled,
+      pluginBeingUninstalled,
+      install,
+      uninstall,
+      onTransitionEnd,
+      tab,
+      selectTab,
+      npmError,
+      fetchFromNpm
+    } = this.props;
+
+    return (
+      <Category>
+        <div className="container">
+          <nav className="plugins-nav">
+            <div
+              className={tab === 'discover' ? 'selected' : ''}
+              onClick={() => selectTab('discover')}
+            >
+              Discover
+            </div>
+            <div
+              className={tab === 'installed' ? 'selected' : ''}
+              onClick={() => selectTab('installed')}
+            >
+              Installed
+            </div>
+          </nav>
+          <div className="tab-container">
+            <div className="switcher"/>
+            <div className="tab">
+              {
+                npmError ? (
+                  <EmptyTab
+                    title="Oops!"
+                    subtitle="Something went wrong…"
+                    link="Refresh"
+                    onClick={fetchFromNpm}/>
+                ) : (
+                  <Tab
+                    current={pluginBeingInstalled}
+                    plugins={pluginsFromNpm}
+                    disabled={Boolean(pluginBeingInstalled)}
+                    onClick={install}/>
+                )
+              }
+            </div>
+            <div className="tab">
+              {
+                pluginsInstalled.length === 0 ? (
+                  <EmptyTab
+                    title="No plugins, yet!"
+                    subtitle="Customize Kap your liking with Plugins."
+                    link="Discover"
+                    onClick={() => selectTab('discover')}/>
+                ) : (
+                  <Tab
+                    checked
+                    disabled={Boolean(pluginBeingInstalled)}
+                    current={pluginBeingUninstalled}
+                    plugins={pluginsInstalled}
+                    onTransitionEnd={onTransitionEnd}
+                    onClick={uninstall}/>
+                )
+              }
+            </div>
+          </div>
+        </div>
+        <style jsx>{`
+          .container {
+            height: 100%;
+            width: 100%;
+            display: flex;
+            flex-direction: column;
+          }
+
+          .plugins-nav {
+            height: 4.8rem;
+            padding: 0 16px;
+            display: flex;
+            align-items: center;
+            box-shadow: 0 1px 0 0 #ddd, inset 0 1px 0 0 #fff;
+            z-index: 10;
+          }
+
+          .plugins-nav div {
+            margin-right: 16px;
+            height: 100%;
+            display: flex;
+            align-items: center;
+            justify-content: center;
+            padding-bottom: 2px;
+            font-size: 1.2rem;
+            color: #007AFF;
```

Comment:
Use lowercase HEX for consistency.

Reply:
`prettier` does this automatically, should we add it?

Reply:
I usually use `xo --fix` if I have lots of errors, and it takes care of most of them

Reply:
But it doesn't format the hex colors.

Reply:
We can discuss pros/cons about adding prettier, but I think we can do that in a different Issue/PR 👍